abd-clam/Cargo.toml

```diff
@@ -43,15 +43,17 @@ priority-queue = "1.3.2"
 # TODO: Break CHAODA out into an optional feature
 # automl = { version = "0.3.0", path = "../rust-automl" }
 # automl = { git = "https://github.com/nishaq503/rust-automl", branch = "updates/dependencies" }
+automl = "0.2.7"
```

Comment:
Add a TODO here for us to switch from `automl` to `smartcore`

Reply:
done